code/game/gamemodes/heist/tools.dm

```diff
@@ -0,0 +1,113 @@
+/**
+ * Handy-Dandy Organ Remover Doohicky
+ * SHOULD ONLY BE MAPPED ON VOX RAIDER SHIPS.
+ */
+/obj/item/weapon/organ_remover
+	name = "organics extractor"
+	desc = "A highly sophisticated tool made for alien hands."
+	icon='icons/obj/surgery.dmi'
+	icon_state = "organ_remover"
+	item_state = "electronic"
+	flags = FPRINT
+	siemens_coefficient = 1
+	slot_flags = SLOT_BELT
+	throwforce = 3
+	w_class = W_CLASS_TINY
+	throw_speed = 5
+	starting_materials = list(MAT_IRON = 200)
+	w_type = RECYK_ELECTRONIC
+	melt_temperature = MELTPOINT_PLASTIC
+	origin_tech = Tc_MAGNETS + "=1;" + Tc_BIOTECH + "=1" // TODO: figure out appropriate values
+
+	var/delay=15 SECONDS // Adminbus purposes.
+	var/vox_only=TRUE    // Same
+	var/list/valid_targets=list(
+		// "brain", Brains have special snowflake shit so this won't work.
+		"eyes",
+		"heart",
+		"kidneys",
+		"liver",
+		"lungs"
+	)
+	var/target_type="eyes"
+
+/obj/item/weapon/organ_remover/examine(var/mob/user)
+	// Everyone except vox knows what these are.
+	if(ishuman(user))
+		var/mob/living/carbon/human/H
+		if(isvox(H))
+			to_chat(user, "A sophisticated device used by vox raiding parties to remove organs without time-consuming surgical procedures.")
+			to_chat(user, "Subject must be still and incapacitated. Remember to set target organs before use!")
+	to_chat(user, "Some weird alien thing, doesn't look like it'd even fit in human hands.")
```

Comment:
Actually, if you look at the flow, the idea is that if it fails to be *either* a human or vox, it then runs that last line.  So it needed a `return` after the "subject must be still" line.